src/index.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable max-len */
+/* eslint-disable no-useless-concat */
+/* eslint-disable no-use-before-define */
+/* eslint-disable radix */
+/* eslint-disable no-undef */
```

Comment:
I notice you disable a lot of Eslint rules here. These rules reinforce good practice and make your code looks better. It also inspires confidence that the code is functional.